components/forward_post_modal/forward_post_channel_select.tsx

```diff
@@ -38,7 +38,7 @@ import {getBaseStyles} from './forward_post_channel_select_styles';
 
 const AsyncSelect = require('react-select/lib/Async').default as React.ElementType<AsyncSelectProps<ChannelOption>>; // eslint-disable-line global-require
 
-type ProviderResults = {
+export type ProviderResults = {
```

Comment:
please move this to the provider file and import here ... also this should probably be renamed to `ProviderResult`